### ISSUE TITLE
Had an idea for a bot, please help make it true!

### ISSUE THREAD

**Issue by aler1337** [binary-com/binary-bot#2546]
Hello everyone!
I've gotten into Binary cus of a guy I follow on IG and he's been selling this bot that does Even/Odd Digits.

Altough I found some online that work with the same strategy, I wanted to edit them so that it would work something like this:

Start with Even or Odd, doesn't matter
 - Example - Start with even, Its a WIN - Change to ODD.
- ODD - It loses, keep on ODD until it wins, then after it wins (with martingale 2x) change to even.

Basically I want the bot to keep using even or odd until it wins while using martingale to double up everytime it loses, and then after it finally wins, change to even or odd, depending on what the last win was.

**Comment by aler1337** [comment]
Either that or EVEN EVEN - ODD ODD - EVEN EVEN - ODD ODD, but I would prefer it to stick to whatever it used until it is a win.

**Comment by ZDEGLET** [comment]
I SAW ON THE WORK 17 TIMES OF Odd CONTRACTS. WELL THINK BEFORE HOW TO WORK ON THIS STRATEGY.

**Comment by aler1337** [comment]
> I SAW ON THE WORK 17 TIMES OF Odd CONTRACTS. WELL THINK BEFORE HOW TO WORK ON THIS STRATEGY.

I understand, do you have any bot like this?

**Comment by aler1337** [comment]
> I SAW ON THE WORK 17 TIMES OF Odd CONTRACTS. WELL THINK BEFORE HOW TO WORK ON THIS STRATEGY.

Or any good bot you can share for Even/Odd?

**Comment by Martinez222** [comment]
@aler1337 this is the bot u want.This is not a good strategy ,but feel free to develop it.
[Prediction EVEN.zip](https://github.com/binary-com/binary-bot/files/4328862/Prediction.EVEN.zip)

**Comment by Kosta92** [comment]
This is one of the worst "strategies" in existence. It's not even a strategy, it's just based on luck, and let me tell you, you'll run out of luck using it and lose your money easily.

**Comment by Martinez222** [comment]
Is what he ask...
